sdk/cosmos/azure-cosmos/azure/cosmos/_change_feed/change_feed_utils.py

```diff
@@ -81,7 +81,7 @@ def validate_kwargs(
     The values of keyword arguments must match the expect type and conditions. If the conditions do not match,
     errors will be raised with the error messages and possible ways to correct the errors.
 
-    :param kwargs: Keyword arguments to verify for query_items_change_feed API
+    :param dict[str, Any] kwargs: Keyword arguments to verify for query_items_change_feed API
```

Comment:
We shouldn't document kwargs like this - this whole line should be removed.
Instead - we need to document each item that we expect to find in kwargs.
"kwargs" itself isn't a parameter, it just indicates that other parameters are expected.

Reply:
Aaaaah sorry that's my bad! Other teams I have reviewed for have documented `**kwargs` as a parameter in that way and I jumped to conclusions without looking close enough - my apologies!